### PR TITLE
[website_forum] Archive does not really archive

### PR DESCRIPTION
**Pull request by hiralvbhavsar**
Description of the issue/feature this PR addresses:

Task: https://www.odoo.com/web?#id=33955&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad: https://pad.odoo.com/p/r.ce8987768714b9d9878c163d04e5d65f

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_forum/models/forum.py [hunk 1]
```diff
@@ -149,12 +149,20 @@ def _compute_count_flagged_posts(self):
         domain = [('forum_id', '=', self.id), ('state', '=', 'flagged')]
         self.count_flagged_posts = self.env['forum.post'].search_count(domain)
 
+    @api.multi
+    def _compute_website_url(self):
+        super(Forum, self)._compute_website_url()
+        for forum in self:
+            forum.website_url = "/forum/%s-%s" % (forum.name, forum.id)
```
Comment (tivisse) [inline addons/website_forum/models/forum.py]: Use slug instead.

File: addons/website_forum/views/website_forum.xml [hunk 2]
```diff
@@ -218,6 +226,19 @@
     </t>
 </template>
 
+<template id="website.403">
```
Comment (tivisse) [inline addons/website_forum/views/website_forum.xml]: Don't overwrite the 403 template if it's to erase it. 

Create another one instead.

Comment (hbh-odoo) [inline addons/website_forum/views/website_forum.xml]: As per the spec we should display the different message for admin and other users that's why we overwrite it. If we create another one then beacuse of access rule the '403: Forbidden' comes for other users.

**Comment by dja-odoo** [comment]
LGTM

**Comment by hmo-odoo** [comment]
partially merged at https://github.com/odoo/odoo/commit/5cafed7fab4492211e9aa32daa5f55e73f3e8390

**Pull request closed without merging**